docs/systemd.md

```diff
@@ -0,0 +1,55 @@
+---
+id: systemd
+title: systemd configuration
+---
+
+To manage starting, logging, etc. of Gotify, systemd can be used.
+
+# Installation
+
+For this example, Gotify is installed by root in a directory called
+`/opt/gotify/` and the executable in there is called `gotify`. The config file
+will be `/etc/gotify/config.yml` using relative paths as in the example and the
+service will be run as root. A file called `/opt/gotify.service` should be
+created by root containing:
+
+```desktop
+[Unit]
+Description=Gotify
+Requires=network.target
+After=network.target
+
+[Service]
+Type=simple
+User=root
```

Comment:
It would be probably better to run gotify as non-root user?

Reply:
Yes, but to run on port 80 or 433, that user needs root rights. So we leave this up to the administrator installing this.

Reply:
You can grant the systemd service permission to open privileged ports without being root. See https://stbuehler.de/blog/article/2017/06/23/systemd__allow_normal_process_to_bind_to_privileged_port.html

This is one of the reasons why there isn't a systemd config in the docs yet, because there is no one size fits all. I'd expect most users to use a reverse proxy before gotify, so it'd be perfectly fine for gotify not to run on a privileged port.

Reply:
Understand. But this PR is a good starting point for those users that don't do that or first want to test it, and need an example. Okay in this context?

Reply:
(That is why it is called a _minimal_ example)